ff/src/fields/mod.rs

```diff
@@ -344,6 +345,32 @@ pub trait PrimeField:
     /// Returns the underlying representation of the prime field element.
     fn into_repr(&self) -> Self::BigInt;
 
+    /// Reads bytes in big-endian, and converts them to a field element.
```

Comment:
Why big-endian? AFAIK most APIs are usually little-endian (for example: `BigInteger` itself

Reply:
(maybe we should add `from_bytes_be_mod_order` and `from_bytes_le_mod_order`.

Reply:
The two hash to field API's I looked at for reference both used big endian, but I applied the suggested change.